examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -434,6 +435,38 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             // unrecognized map type
             break;
         }
+      },
+
+      pinMapOneTweetQuery: function(pinid){
```

Comment:
1. Name change: pinMapOneTweetLookUpQuery
2. Please put a comment about this query. 
// This query retrieves the information about a specific tweet that the user selected.